testing/scenario_app/lib/src/locale_initialization.dart

```diff
@@ -66,4 +68,42 @@ class LocaleInitialization extends Scenario {
       )).build()
     );
   }
+
+  /// Handle taps.
+  ///
+  /// Send changing information via semantics on each successive tap.
+  @override
+  void onPointerDataPacket(PointerDataPacket packet) {
+    String label;
+    switch(tapCount) {
+      case 0: label = window.platformResolvedLocale.toString(); break;
```

Comment:
nit: I find this syntax hard to read even though it's shorter, I'd prefer:
```suggestion
      case 0: {
        label = window.platformResolvedLocale.toString();
      }
      break;
```